missions/LCU_Missions.txt

```diff
@@ -47,25 +40,19 @@ lcu_slot1_branch = {
 		required_missions = { LCU_prepare_for_war }
 		position = 2
 		provinces_to_highlight = {
-			OR = {
-				area = Northern_moon_state
-				area = Uncolonized_moon_state
-			}
+			region = moon_region
 			NOT = { country_or_non_sovereign_subject_holds = ROOT }
 		}
 		
 		trigger = {
 			num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
-				OR = {
-					area = Northern_moon_state
-					area = Uncolonized_moon_state
-                   }
-				value = 4
+				region = moon_region
+				value = 300 # start with 251
 			}
 		}
 		effect = {			
 			add_adm_power = 100
```

Comment:
Considering that the new map is bigger, we probably need to increase the reward to something like
add_adm_power = 300

Reply:
300 seems a bit high of one mana type perhaps splitting it into 2 or 3 types might be better so 150 of admin and dip or 100 of all three?

Reply:
I think idea is to offset some cost of conquest. I don't think a mil mana reward makes sense here, but an increase in admin and maybe some diplo can be fine.

Reply:
fair point, changed to 300